ntoskrnl/mm/mminit.c

```diff
@@ -127,46 +127,83 @@ MiDbgDumpAddressSpace(VOID)
     //
     DPRINT1("          0x%p - 0x%p\t%s\n",
             KSEG0_BASE,
-            (ULONG_PTR)KSEG0_BASE + MmBootImageSize,
+            (ULONG_PTR)KSEG0_BASE + MmBootImageSize - 1,
```

Comment:
What if the convention of the displayed ranges were `[start, end)` (or as we write in french maths, `[start, end[`) ?

Reply:
For example, 'end included' avoids the `0xFFFFFFFF + 1` overflow.
Nonetheless, I agree 'end excluded' could be good enough.
The main point is not to mix the two as current code does.